### ISSUE TITLE
DateRangePicker Accessibility Issue

### ISSUE THREAD

**Issue by ankush-sood-tech** [mymth/vanillajs-datepicker#168]
I am using VanillaJs-datepicker library 1.3.3 version in my project and i was working on the accessibility issue we found in this for date range calendar.

Steps to reproduce the issue :-

1. Open a web page that uses the vanillajs-datepicker library.
2. Navigate to the dateRangePicker input field using the "Tab" key on your keyboard.
3. Once the input field has focus, the entire calendar will be read out by the screen reader.
4. Use the arrow keys on your keyboard to navigate through the calendar cells/spans.
5. Notice that the screen reader does not pronounce the date when you move through the cells/spans using the arrow keys. Instead, it pronounces the word "blank".
6. If you select a date from the next month, the screen reader will read out the entire month's dates at once, which can be overwhelming for users who rely on screen readers.
7. Also, The screen reader does not read out the names of the days of the week, only the short forms (e.g. "Su", "Mo", "Tu", etc.), which can make it difficult for users to understand which day they are selecting.

![image](https://github.com/mymth/vanillajs-datepicker/assets/148582584/f8d60c9a-4dcd-4f0c-b0a1-3661db10c663)
